drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -172,6 +172,13 @@ class DragAndDropBlock(XBlock, XBlockWithSettingsMixin, ThemableXBlockMixin):
     block_settings_key = 'drag-and-drop-v2'
     has_score = True
 
+    def max_score(self):  # pylint: disable=no-self-use
+        """
+        Return the problem's max score, which for DnDv2 always equals 1.
```

Comment:
Note: Currently, you are defining the `max_score` as 1.  However, in the future, you may consider having the `max_score` be equal to the number of items that need to be dragged.  This way, when DND is used as an assessment, learners can get partial credit for some of the items they got correct.